packages/bcp/src/types.ts

```diff
@@ -16,7 +16,22 @@ export interface TokenInfo {
   readonly fractionalDigits: number;
 }
 
-export type TokenInfos = ReadonlyArray<TokenInfo>;
+export type BankTokens = ReadonlyArray<BankToken>;
+
+export interface Erc20Token {
+  readonly contractAddress: string;
```

Comment:
Besides this address, the rest we can query from the chain, right? Ticker and fractionalDigits 

It would be nice to eventually make the  cosmwasmconnection constructor just take the addresses and query the rest

Reply:
Ideally yes. However, I am not sure what information is needed to configure the codec. Let's see ...

Reply:
This is more an idea for the future when we load some config file with erc20 tokens, or auto-config. Explicit is very nice for tests